boards/arm/mimxrt1050_evk/board.cmake

```diff
@@ -7,5 +7,9 @@
 board_runner_args(pyocd "--target=mimxrt1050_hyperflash")
 board_runner_args(jlink "--device=MCIMXRT1052")
 
+if(${CONFIG_BOARD_MIMXRT1050_EVK_QSPI})
+    board_runner_args(jlink "--loader=BankAddr=0x60000000&Loader=QSPI")
```

Comment:
Would it make sense to use ${FLASH_BASE_ADDRESS} instead of the constant address here?

Reply:
There might be an offset to take into account as well. I have ideas to make this submission generic. However, this implementation addresses the bug. I am working on an improvement that can be thoroughly looked at after the release deadline.
How does this sound?

Reply:
@mmahadevan108 asked that I share these test results, testing this PR on all the NXP i.MX RT boards supported in Zephyr.

This PR was prompted by Segger making a change, and we want to ensure this is backwards compatible.  So I tested with both JLink versions 7.80a after Segger’s change, and 7.70c before the change.  Everything looks good to me.

With the latest v7.80a, the following boards flash and work great for me:
mimxrt1050_evk
mimxrt1050_evk_qspi
mimxrt1060_evk

Note that I also built apps using board mimxrt1060_evk_hyperflash, and flashed to the mimxrt1050_evkb board using the same hyperflash memory.  The flash was programmed and verified successfully, but the Zephyr apps didn’t boot.  But this is good enough for me for this, because it does program correctly, so the booting may be related to using the wrong board.  But I do not have a mimxrt1060_evk modified for HyperFlash.

I also reverted JLink to v7.70c and tested with these default boards:
mimxrt1050_evk
mimxrt1060_evk

I also tested all these boards work with JLink 7.80a without needing any changes to Zephyr tree:
mimxrt595_evk_cm33
mimxrt685_evk_cm33
mimxrt1010_evk
mimxrt1015_evk
mimxrt1020_evk
mimxrt1024_evk
mimxrt1064_evk
mimxrt1160_evk_cm4
mimxrt1160_evk_cm7
mimxrt1170_evk_cm4
mimxrt1170_evk_cm7

So this PR looks great to me.  Thank you, Yves!